packages/virtual-core/src/index.ts

```diff
@@ -158,7 +159,14 @@ const createOffsetObserver = (mode: ObserverMode) => {
       const scrollX = target[propX]
       const scrollY = target[propY]
 
-      if (instance.options.horizontal ? prevX - scrollX : prevY - scrollY) {
+      let shouldScroll;
+      if (instance.options.horizontal) {
+        shouldScroll = instance.options.reverse ? scrollX - prevX : prevX - scrollX;
+      } else {
+        shouldScroll = instance.options.reverse ? scrollY - prevY : prevY - scrollY;
+      }
+
+      if (shouldScroll) {
```

Comment:
Should this be a more clear boolean comparison, like `if (shouldScroll) > 0` ?

Reply:
I don't think it should, it seems that the idea is to scroll in either direction if needed.